ext/dba/dba.c

```diff
@@ -318,7 +318,7 @@ static void dba_close_pe_rsrc(zend_resource *rsrc)
 /* }}} */
 
 /* {{{ PHP_INI */
-ZEND_INI_MH(OnUpdateDefaultHandler)
+static ZEND_INI_MH(OnUpdateDefaultHandler)
```

Comment:
Would it make sense to modify the definition of ``ZEND_INI_MH`` to include the ``static``?

Reply:
I thought about that, but after I saw it's already used that way a lot (54 times), I decided it's a whole different refactoring case and didn't check if it breaks anything. e.g.:
https://github.com/php/php-src/blob/e093ac4b545d61d9e0a5d0eaefe4f480cf22b36f/Zend/zend.c#L101

Reply:
Huh, didn't even realize that we were already doing this elsewhere. Makes sense then to just add it here